pkg/util/strings.go

```diff
@@ -32,6 +32,14 @@ func SplitString(str string) []string {
 	if len(str) == 0 {
 		return []string{}
 	}
+	if strings.Contains(str, ",") {
```

Comment:
since this function is used in several placed I would suggest making this functionality optional (introduce a flag parameter disabled by default) and enable it only for the specific cases
WDYT?

Reply:
It adds some extra complexity. So I want to make sure we do not use it to handle cases like:
```
"foo, bar baz" => ["foo", "bar", "baz"]
```

If we don't, I prefer to keep it simple.

Reply:
ok

Reply:
Maybe I can add separate method for handling configuration options, like `SplitConfigParams()` to reduce affected area.

Reply:
this comma separation seems dangerous for LDAP groups, I've added a comment about specific scenarios